docs/platforms/python/troubleshooting.mdx

```diff
@@ -136,3 +136,27 @@ sentry_sdk.init(
 
 If you need more fine-grained control over the behavior of the socket, check out
 <PlatformLink to="/configuration/options/#socket-options">socket-options</PlatformLink>.
+
+## Multiprocessing deprecation after Python 3.12
+
+If you're on Python greater than 3.12, you might see the following deprecation warning on Linux environments since the SDK spawns several threads.
+
+```
+DeprecationWarning: This process is multi-threaded, use of fork() may lead to deadlocks in the child.
```

Comment:
```suggestion
DeprecationWarning: This process is multi-threaded, use of fork() may lead to deadlocks in the child process.
```

Reply:
this is a deprecation warning from python, that's how they have it